gapc_storage/storage.py

```diff
@@ -38,9 +72,12 @@ class GoogleCloudStorage(Storage):
     makes no assumptions about your environment and can be used anywhere.
     """
 
+    path_prefix = ""
+
     def __init__(self):
         self.set_client()
         self.bucket = _gcs_file_storage_settings()["bucket"]
+        self.path_prefix = self.path_prefix.lstrip("/")
```

Comment:
@brosner: I'm trying to think through how overrides would work if you had two `GoogleCloudStorage` backend instances in a project (one for media, one for statics). 

I'm proposing the following resolution order:
- use the `path_prefix` defined on a subclass
- if the subclass has no `path_prefix`, fall back to `GAPC_STORAGE[path_prefix]`
- the default for `GAPC_STORAGE[path_prefix]` is an empty string